cfg/cspell-dictionary.txt

```diff
@@ -1,10 +1,17 @@
+LIGHTBLACK
+LIGHTGREEN
+LIGHTMAGENTA
+LIGHTRED
+LIGHTWHITE
+LIGHTYELLOW
```

Comment:
I think these should be escaped in the individual Python file

Reply:
ok will do, I wasn't sure since I thought I might add more scripts using them, but probably better this way to avoid cluttering the cspell dictionary